File: php/testcode/benchmark_test_00566.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00566(BenchmarkRequest $req): BenchmarkResponse {
    $path = $req->param('path');
    $exists = file_exists("phar://" . $path);
    return BenchmarkResponse::ok($exists ? 'found' : 'not found');
}

File: rust/testcode/benchmark_test_00566.rs
pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let action = req.param("action");
    let db_password = secrets_manager_get("app/db-password");
    let result = format!("Action {} with Secrets Manager credential", action);
    super::shared::BenchmarkResponse::ok(&result)
}

fn secrets_manager_get(secret_id: &str) -> String {
    format!("secret_{}", secret_id)
}
